### ISSUE TITLE
REST API wing lib

### ISSUE THREAD

**Issue by tsuf239** [winglang/wing#5384]
### Use Case

To create a REST API for multiple tables, within a single resource - to eliminate the amount of redundant wing code and time needed to deploy one

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.
* If this issue is labeled **needs-discussion**, it means the spec has not been finalized yet. Please reach out on the #dev channel in the [Wing Slack](https://t.winglang.io/slack).

**Comment by staycoolcall911** [comment]
@tsuf239 - can you please add more details about the use case here?

**Comment by tsuf239** [comment]
Yes, I can describe what it should solve without getting into implementation details-
1 - The resource should take a list of tables/schemas
2 - for each schema, it should create a full REST API  (GET, PUT/PATCH, POST DELETE routes) 
3 - We need to think about how to handle foreign keys and bulk operations
4 - We should also take authentication and authorization into consideration
5 - We should think about how flexible our model is- what overrides do we allow? etc...

**Comment by staycoolcall911** [comment]
Thank you 🙏
